pages/docs/create-new-form.mdx

```diff
@@ -2,46 +2,31 @@
 
 ## Introduction
 
-When creating user interface forms in OpenMRS frontends, the AMPATH form builder platform makes this task much easier by partially automating the process.
+The React Form Engine (RFE) form builder platform simplifies and partially automates the process of generating user interface forms in OpenMRS frontends.
 
-## Important links/concepts used in this document:
+## Links and ideas that are important to this text:
 
-- Getting started - https://ampath-forms.vercel.app/getting-started
-- Core concepts - https://ampath-forms.vercel.app/core-concepts
-- Validating fields - https://ampath-forms.vercel.app/validating-fields
-- Field types - https://ampath-forms.vercel.app/field-types-reference
+- Getting started - https://ohri-docs-chi.vercel.app/docs/create-new-form
+- Core concepts - https://ohri-docs-chi.vercel.app/docs/core-concepts/forms
+- Validating fields -https://ohri-docs-chi.vercel.app/docs/validation
+- Field types - https://ohri-docs-chi.vercel.app/docs/field-types-reference
+- Developer Guide - https://ohri-docs-chi.vercel.app/docs/developer-guide/contributing-guide
 
-## What is the AMPATH form builder?
+## What is the RFE form builder?
 
-The AMPATH form builder found at the address: https://openmrs-spa.org/formbuilder is an automation tool/website for auto-generating form schema JSON files which can be later loaded into the Form Engine to auto-create micro-frontend user interface forms.
+The RFE form builder found at the address: https://ohri-dev.globalhealthapp.net/openmrs/spa/login is an automation tool/website for auto-generating form schema JSON files which can be later loaded into the Form Engine to auto-create micro-frontend user interface forms.
```

Comment:
This is the correct path for the form builder within 
-> OHRI https://ohri-dev.globalhealthapp.net/openmrs/spa/form-builder 
-> RefApp https://dev3.openmrs.org/openmrs/spa/form-builder

Reply:
Noted and updated